examples/Agent-Demos/ComputerUse-Agent-Demo/image/static_content/index.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html>
+<head>
+    <title>Computer Use Demo</title>
+    <meta name="permissions-policy" content="fullscreen=*"/>
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+            overflow: hidden;
+        }
+        .container {
+            display: flex;
+            height: 100vh;
+            width: 100vw;
+        }
+        .left {
+            flex: 1;
+            border: none;
+            height: 100vh;
+        }
+        .right {
+            flex: 2;
+            border: none;
+            height: 100vh;
+        }
+    </style>
+    <script>
+        function hideLoadingMessage() {
+            document.getElementById('loading-message').style.display = 'none';
+            const iframe = document.getElementById('my-iframe');
+            const loadingMessage = document.getElementById('loading-message');
+            loadingMessage.style.display = 'none';
+            iframe.style.display = 'block';
+        }
+
+        function showErrorMessage() {
+          document.getElementById('loading-message').textContent = 'Error: URL not accessible.';
+        }
+
+        // Optional: Timeout for handling cases where iframe hangs
+        setTimeout(() => {
+          const iframe = document.getElementById('my-iframe');
+          if (!iframe.contentDocument || iframe.contentDocument.readyState !== 'complete') {
+            showErrorMessage();
+          }
+        }, 5000);  // 5-second timeout
+    </script>
+</head>
+<body>
+<div class="container">
+    <p id="loading-message">Loading... Please wait.</p>
+    <iframe
+            id="my-iframe"
+            src="http://localhost:7860"
+            class="left"
+            allow="fullscreen"
+            onload="hideLoadingMessage()"
+            onerror="showErrorMessage()"
+    ></iframe>
```

Comment:
This stuff is only used from within the docker image correct? Meaning it should be safe to dismiss this?